chatroom/src/components/SingleCard.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Button from 'react-bootstrap/Button';
+import Card from 'react-bootstrap/Card';
+
+function SingleCard(props) {
+  return (
+    <div>
+      <Card style={{ width: "18rem" }}>
```

Comment:
Stylings can be done by using a css file and using className="" in here instead of hardcoding style elements here